packages/svelte/src/runtime/internal/dom.js

```diff
@@ -399,6 +399,15 @@ export function self(fn) {
 	};
 }
 
+/**
+ * @returns {(event: any) => void} */
```

Comment:
this formatting is not consistent with the rest of the codebase

Reply:
It's consistent with all of the other event modifiers in this file...

I'll try to find some time to improve it